plugins/simpleadminhacks-paper/src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/BuildLimits.java

```diff
@@ -0,0 +1,65 @@
+package com.programmerdan.minecraft.simpleadminhacks.hacks;
+
+import com.programmerdan.minecraft.simpleadminhacks.SimpleAdminHacks;
+import com.programmerdan.minecraft.simpleadminhacks.configs.BuildLimitsConfig;
+import com.programmerdan.minecraft.simpleadminhacks.framework.SimpleHack;
+import com.programmerdan.minecraft.simpleadminhacks.framework.utilities.BuildLimit;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.NamedTextColor;
+import org.bukkit.Location;
+import org.bukkit.configuration.ConfigurationSection;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
+import org.bukkit.event.Listener;
+import org.bukkit.event.block.BlockPlaceEvent;
+import org.jetbrains.annotations.NotNull;
+
+public class BuildLimits extends SimpleHack<BuildLimitsConfig> implements Listener {
+	public BuildLimits(SimpleAdminHacks plugin, BuildLimitsConfig config) {
+		super(plugin, config);
+	}
+
+	public static BuildLimitsConfig generate (
+			final @NotNull SimpleAdminHacks plugin,
+			final @NotNull ConfigurationSection config){
+		return new BuildLimitsConfig(plugin, config);
+	}
+
+	@Override
+	public void onEnable(){
+		plugin.registerListener(this);
```

Comment:
unnecessary

Reply:
This one actually is necessary: it's only BasicHack that registers itself as a listener upon `onEnable`

Reply:
goddamnit i hate hacks